### PR TITLE
Added row about OST and PST files

### PR DESCRIPTION
**Pull request by MaratMussabekov**
#6172 
#6343

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9, could you please have a look?
Thank you

File: windows/security/information-protection/windows-information-protection/limitations-with-wip.md [hunk 1]
```diff
@@ -132,6 +132,14 @@ This table provides info about the most common problems you might encounter whil
         <td>If you need to save an encrypted file in the Windows directory, create and encrypt the file in a different directory and copy it.
         </td>
     </tr>
+    <tr>
+        <td>Microsoft Office Outlook offline data files (PST and OST files) are not marked as <strong>Work</strong> files, and therefore are not protected.
+        </td>
+        <td>If Microsoft Office Outlook is set to work in cached mode (default setting), or if some emails are stored in local PST file, the data is unprotected.
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/windows-information-protection/limitations-with-wip.md]:
```suggestion
        <td>If Microsoft Office Outlook is set to work in cached mode (default setting), or if some emails are stored in a local PST file, the data is unprotected. 
```

File: windows/security/information-protection/windows-information-protection/limitations-with-wip.md [hunk 2]
```diff
@@ -132,6 +132,14 @@ This table provides info about the most common problems you might encounter whil
         <td>If you need to save an encrypted file in the Windows directory, create and encrypt the file in a different directory and copy it.
         </td>
     </tr>
+    <tr>
+        <td>Microsoft Office Outlook offline data files (PST and OST files) are not marked as <strong>Work</strong> files, and therefore are not protected.
+        </td>
+        <td>If Microsoft Office Outlook is set to work in cached mode (default setting), or if some emails are stored in local PST file, the data is unprotected. 
+        </td>
+        <td>It is recommended to use Microsoft Office Outlook in Online mode, or use encryption to protect OST and PST files manually.
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/windows-information-protection/limitations-with-wip.md]:
```suggestion
        <td>It is recommended to use Microsoft Office Outlook in Online mode, or to use encryption to protect OST and PST files manually.
```

File: windows/security/information-protection/windows-information-protection/limitations-with-wip.md [hunk 3]
```diff
@@ -132,6 +132,14 @@ This table provides info about the most common problems you might encounter whil
         <td>If you need to save an encrypted file in the Windows directory, create and encrypt the file in a different directory and copy it.
         </td>
     </tr>
+    <tr>
+        <td>Microsoft Office Outlook offline data files (PST and OST files) are not marked as <strong>Work</strong> files, and therefore are not protected.
```
Comment (illfated) [inline windows/security/information-protection/windows-information-protection/limitations-with-wip.md]: "therefore" is used as an adverb here, it is therefore more common to add it after the verb in a sentence clause.
```suggestion
        <td>Microsoft Office Outlook offline data files (PST and OST files) are not marked as <strong>Work</strong> files, and are therefore not protected.
```

**Comment by illfated** [comment]
> files, so those files are not provided..
> 
I think you meant "protected", but OK, I got the gist of it.

> Above line makes sense

Yes, it sure does. Knowing that the Outlook file (`.OST` or `.PST`) containing all the e-mail messages is in fact a type of database file, it becomes more obvious that you can not treat that file as a document.

**Comment by mypil** [comment]
@DulceMontemayor @Dansimp - PR has been copy edited and is ready for final review. This is based on issue #6172 and #6343.

Thank you.

cc: @kenwith

**Pull request merged**